src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  padding-top: 12px;
+}
+
+.field {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 900;
```

Comment:
should be 300 by default and 900 on focus

Reply:
done